### ISSUE TITLE
shuffle bn for voxel input

### ISSUE THREAD

**Issue by AndyYuan96** [facebookresearch/DepthContrast#18]
hi，zaiwei，I see that in the code you don't use shuffle bn when input is voxel，it's just the experiment result or some idea behind the operation？

**Comment by AndyYuan96** [comment]
> I tried the shuffle bn for voxel but did not notice any performance gain. It probably makes the results worse. That's why I disabled it for now. For outdoor 3d object detection, I was trying to pick a model that uses voxel as input, and performs well during my submission. That's why I choose partA2. Pointpillar uses a mixed type of representation and performs slight worse than PartA2. That's why I did not pick it.

what do you mean “worse”，for pointpillar， does the improvement that using pretrained model compared to training from scratch， is less than partA2？

**Comment by zaiweizhang** [comment]
"slight worse" is referring to training from scratch. I have not tried using pretrained model with Pointpillar.

**Comment by AndyYuan96** [comment]
> "slight worse" is referring to training from scratch. I have not tried using pretrained model with Pointpillar.

OK, I will try to use pointpillar as base model, to make training more faster, and I will try on Waymo dataset, for domain adaptation is a big question in lidar dataset, I prepare to only use Waymo dataset as pretrain dataset and fine-tune dataset to see if moco style self supervised learning works.

**Comment by zaiweizhang** [comment]
The fine-tuning will happen on KITTI?

**Comment by zaiweizhang** [comment]
Closing. Feel free to open if you have new questions.
